relays/ethereum/src/substrate_types.rs

```diff
@@ -88,7 +88,7 @@ pub fn into_substrate_ethereum_header(header: &EthereumHeader) -> SubstrateEther
 		extra_data: header.extra_data.0.clone(),
 		state_root: header.state_root,
 		receipts_root: header.receipts_root,
-		log_bloom: header.logs_bloom.data().into(),
+		log_bloom: header.logs_bloom.unwrap_or_default().data().into(),
```

Comment:
Probably worth checking this when header is received (`header_by_number` and `header_by_hash` in `ethereum_client.rs`). There's similar check for hash and number. This isn't critical, though

Reply:
Are you talking about adding a check [here](https://github.com/paritytech/parity-bridges-common/blob/master/relays/ethereum/src/ethereum_client.rs#L156)?

If so, I'm not sure I understand. Not having a `log_bloom` shouldn't be reason to mark a header an incomplete, right?

Reply:
I guess 'incomplete' word now has too many meanings in relay :/ In this check, incomplete means that Ethereum node haven't returned all the fields we use. Since we use lob bloom (we are passing it to the runtime module), then yes - it should fail with `Error::IncompleteHeader` as well.

That differs from 'incomplete' that has been introduced recently, which is about some state of the header at the receiving side.